worker/src/utils/group-versions.ts

```diff
@@ -0,0 +1,17 @@
+export const groupVersions = (versions: Record<string, number>) => {
+  const releases: Record<string, number> = {};
+  const releases_filtered: Record<string, number> = {};
+
+  Object.keys(versions).forEach((version) => {
+    const key: string = version.split(".").slice(0, 2).join(".");
+    releases[key] = (releases[key] || 0) + versions[version];
+  });
+
+  Object.keys(releases)
+    .filter((key) => releases[key] > 100)
```

Comment:
Why are releases with < 100 excluded?

Reply:
The core idea behind this PR is to be able to show changes over time, e.g. how new versions are adopted.
It is not really useful to keep versions around that only have some residual users; compared to any other version they won't matter. Additionally, this will also reduce the data size of each entry as only the relevant information is stored.